xbmc/platform/android/activity/XBMCApp.h

```diff
@@ -258,6 +258,7 @@ class CXBMCApp : public IActivityHandler,
   bool m_wakeUp{false};
   bool m_aeReset{false};
   bool m_hdmiPlugged{true};
+  unsigned int m_MediaSessionVideoUpdateCounter = 0;
```

Comment:
We usually start member variable names lower case after the m_ prefix.

Reply:
I will fix that immediately. Actually I am sitting in the sun ... and was happy that I produced something that looked like kodi code - with just 300 NITs.